.github/workflows/e2eTest.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           cache-read-only: false
 
       - name: Start Docker Solution
-        run: ./gradlew dockerSolution:ComposeUp -x test
+        run: ./gradlew dockerSolution:ComposeUp -x test --scan --info --stacktrace
```

Comment:
Is --info bumping the level just for scan or for all output?  If it's for all output, do you know how much bigger the logfiles will grow?

Reply:
It's only for the Gradle specific logs (like reason for task cache ignored), minimally impacts the size